modules/vaos/app/services/vaos/appointment_service.rb

```diff
@@ -17,6 +17,15 @@ def get_appointments(type, start_date, end_date, pagination_params = {})
       end
     end
 
+    def get_appointment(id)
+      params = {}
+
+      with_monitoring do
+        response = perform(:get, show_appointment_url(id), params, headers, timeout: 55)
```

Comment:
Get appointment with an ID should return much faster than get appointments, so lets not pass a timeout at all and use default. 

Lets also open a ticket to investigate latency issues with get_appointments, because the present value of 55 appears extreme.